es-app/src/CollectionSystemManager.cpp

```diff
@@ -169,7 +142,8 @@ void CollectionSystemManager::updateSystemsList()
 	// remove all Collection Systems
 	removeCollectionsFromDisplayedSystems();
 	// add custom enabled ones
-	addEnabledCollectionsToDisplayedSystems(&mCustomCollectionSystemsData);
+	//addEnabledCollectionsToDisplayedSystems(&mCustomCollectionSystemsData);
+	addEnabledCollectionsToDisplayedSystems();
```

Comment:
I don;t think this method actually needs an argument, as we are passing it a member function anyways.

Reply:
Thanks for catching that. In the beginning that wasn't the case, but after a few iterations I suppose it ended up like that! Good catch.